Source/WebKit/WebProcess/WebProcess.messages.in

```diff
@@ -209,4 +209,6 @@ messages -> WebProcess LegacyReceiver NotRefCounted {
 #if PLATFORM(MAC)
     OpenDirectoryCacheInvalidated(WebKit::SandboxExtension::Handle handle)
 #endif
+
+    permissionChanged(enum:uint8_t WebCore::PermissionName permissionName, struct WebCore::SecurityOriginData topOrigin)
```

Comment:
Ideally, this IPC would be on WebPermissionController, not on WebProcess. We have a dedicated class for this so we shouldn't pollute the generic WebProcess. You likely want to introduce a WebPermissionController.messages.in instead.

You can use WebLockRegistryProxy as reference.